src/components/PeopleFilters/PeopleFilters.tsx

```diff
@@ -0,0 +1,123 @@
+import classNames from "classnames";
+import { FILTER } from "../../types/filters";
+import { Link } from "react-router-dom"; 
+
+type Props = {
+  filterSex: FILTER,
+  handleSexFilterChange: (sex?: FILTER) => void,
+  query: string,
+  handleQueryChange: React.ChangeEventHandler<HTMLInputElement>,
+  centuries: string[],
+  handleCenturiesFilter: (ch: string) => void,
+  handleSelectAllCenturies: () => void,
+}
+
+export const PeopleFilters: React.FC<Props> = ({ 
+  filterSex, 
+  handleSexFilterChange,
+  query,
+  handleQueryChange,
+  centuries,
+  handleCenturiesFilter,
+  handleSelectAllCenturies,
+}) => {
+  const arrCenturies = [16, 17, 18, 19, 20];
+
+  return (
+    <nav className="panel">
+      <p className="panel-heading">Filters</p>
+
+      <p className="panel-tabs" data-cy="SexFilter">
+        <Link className={classNames({'is-active': filterSex === FILTER.ALL})} to="/people" 
+          onClick={(e) => {
+            e.preventDefault();
+            handleSexFilterChange();
+          }}
+        >
+          All
+        </Link>
+        <Link className={classNames({'is-active': filterSex === FILTER.MALE})} to="/people?sex=m" 
+          onClick={(e) => {
+            e.preventDefault();
+            handleSexFilterChange(FILTER.MALE);
+          }}
+        >
+          Male
+        </Link>
+        <Link className={classNames({'is-active': filterSex === FILTER.FEMALE})} to="/people?sex=f" 
```

Comment:
Similar to the previous comment, ensure that the `to` prop with `?sex=f` is correctly handled in your routing logic to filter the data based on the query parameter.